library/src/main/kotlin/com/livefront/gsonkotlinadapter/KoltinReflectiveTypeAdapterFactory.kt

```diff
@@ -121,8 +140,14 @@ class KotlinReflectiveTypeAdapterFactory private constructor() : TypeAdapterFact
          * Returns an new instance of [KotlinReflectiveTypeAdapterFactory] which constructs classes
          * using the default constructor, allowing for properties to initialized the properly.
          * This ensures that the JSON fulfills the nullability contract of the model and calls the
-         * class's init block.
+         * class's init block. Setting the [enableDefaultPrimitiveValues] to `true` will allow
+         * nonnull, primitive values to use default values when Kotlin default values are not
```

Comment:
I'm not sure the comma after `nonnull` is required here. "primitive values" is kind of a single concept already.

Reply:
Just a typo